serpent-moltres/isotope-distribution/create-xs.sh

```diff
@@ -0,0 +1,38 @@
+# Step 1: Uncomment these lines to create .coe
```

Comment:
What do you mean uncomment these lines? arent they uncommented

Reply:
my bad. good catch.